CHANGELOG.md

```diff
@@ -27,6 +27,8 @@
 - Simplify CLI option handling by only allowing one option at a time to be added ([#54])
 - Redefine OQT regions ([#26])
 - Use ohsome API endpoint `/contributions/latest/count` for Last Edit indicator ([#68])
+- Implement `as_feature` function for indicator and report classes ([#86])
+- Use new `as_feature` function of report and indicator classes in CLI ([#86])
```

Comment:
Don't you want to use the separate PR numbers as used in #87? If not, you can get rid of the second line.

Reply:
I removed the second line. All information all listed in the description of PR #86